skyvern/forge/sdk/workflow/models/block.py

```
@@ -4484,11 +4483,12 @@ class PrintPageBlock(Block):
     format: str = "A4"
     landscape: bool = False
     print_background: bool = True
+    parameters: list[PARAMETER_TYPE] = []
 
     VALID_FORMATS: ClassVar[set[str]] = {"A4", "Letter", "Legal", "Tabloid"}
 
     def get_all_parameters(self, workflow_run_id: str) -> list[PARAMETER_TYPE]:
```

Ruff (0.14.13)

Unused method argument: `workflow_run_id`

(ARG002)